modules/ROOT/pages/changes-feed.adoc

```diff
@@ -31,10 +31,9 @@ include::partial$server-integration-scenario-table.adoc[]
 == Changes Feed
 
 
-The changes-feed for the reliever returns a sorted list of changes made to documents in the database.
+The changes-feed for the reliever returns a sorted list of changes made to documents in the keyspace.
```

Comment:
reliever? is that a term, or do we mean receiver or something else?

Reply:
Let's ask GPT as s first shot